api/api.go

```diff
@@ -72,7 +73,9 @@ func initialiseStubbedAreaData(_ context.Context) (map[string]models.AreasDataRe
 	geoData := make(map[string]models.AreasDataResults, 2)
 	for _, geoDataFile := range fls {
 		var data models.AreasDataResults
-		json.Unmarshal([]byte(geoDataFile), &data)
+		if err := json.Unmarshal([]byte(geoDataFile), &data); err != nil {
+			log.Fatalf("error unmarshalling geo data %+v", err)
```

Comment:
Do we want the service to die here? Shouldn't we instead return an error back from `Setup()` and log using the ONS log package instead?

Reply:
Actually the reason why I was failing this is when I had a. look at the ```initialiseStubbedAreaData``` this seems to be critical i.e if this fails then there seemed to be no point in getting the service up and running. But actually as you mentioned it would be good to return an err rather and let the alling package handle it. Fixed in update 2.